bbot/modules/baddns.py

```diff
@@ -26,10 +26,17 @@ class baddns(BaseModule):
     max_event_handlers = 8
     deps_pip = ["baddns~=1.1.789"]
 
+    per_host_only = True
+    scope_distance_modifier = 1
+
+    def _incoming_dedup_hash(self, event):
```

Comment:
this is not needed since it overrides `per_host_only`

Reply:
resolved